games/ninswitch/game_simple_reset.py

```diff
@@ -1,11 +1,32 @@
 from surrortg import Game
+from surrortg.inputs import Switch
 from games.ninswitch.ns_gamepad_serial import NSGamepadSerial, NSButton, NSDPad
 from games.ninswitch.ns_switch import NSSwitch
 from games.ninswitch.ns_dpad_switch import NSDPadSwitch
 from games.ninswitch.ns_joystick import NSJoystick
+import RPi.GPIO as GPIO
```

Comment:
So far we have used `pigpio` library (http://abyz.me.uk/rpi/pigpio/) instead of Rpi.GPIO, so for consistency I would use it here also.

The interface is a bit clunky, bit it has more advanced features. Example use: https://github.com/SurrogateInc/surrortg-sdk/blob/main/games/claw/game.py